tests/frontend/test_circuitspecs_chip2.py

```diff
@@ -359,6 +359,7 @@ def test_no_unitary(self, tol):
             ops.MeasureFock() | q
 
         res = prog.compile("chip2")
+        res.print()
```

Comment:
Was this change also required due to #311?

Reply:
Nope, this was me debugging, good catch!
```suggestion
```